added zero-to-hero docs for twitter dapp (#86)
dappforce/subsocial-docs: docs/develop/zero-to-hero/twitter-dapp/FinishingApp.md

---
id: finish-app

> title: Finishing Dapp

Reviewer: ```suggestion
title: Finishing The Dapp
```

displayed_sidebar: developSidebar
---

## Congratulations 🥂🎉

Now, you have your very own twitter-like Dapp on Subsocial.

> Share it with people to show-off your skills and learning.

Reviewer: ```suggestion
Share it with people to show off your skills and learning.
```

Reviewer: and maybe just put the below hyperlink onto "Share" in this line and delete line 13

[Share on Twitter](https://ctt.ac/6G7EI)

### What's next? 🤔

> **Subsocial** helps you build the next generation social media in no time. Here are a couple of things to do next:

Reviewer: ```suggestion
**Subsocial** helps you build the next generation of social media in no time. Here are a couple of things to do next:
```

- Experiment more with our SDK using the [Playground](https://play.subsocial.network)

> - Check the hackathons & bounties to build next, [here](/docs/bounties).

Reviewer: ```suggestion
- Check out the [hackathons & bounties](/docs/bounties).
```

> - Connect with us on [Telegram Group](https://t.me/+ZzvLu0ZfkQwxNGQy) and share your experience.

Reviewer: ```suggestion
- Connect with us on [Telegram](https://t.me/+ZzvLu0ZfkQwxNGQy) and share your experience.
```

> > Learn more about different components that make up Subsocial structures called [Entities](/docs/develop/concepts/entities).

Reviewer: ```suggestion
> Learn more about the different components that make up Subsocial's structures called [Entities](/docs/develop/concepts/entities).
```
